cfc-testing/scripts/compliance-report-generator.py

```
@@ -0,0 +1,1260 @@
+
+                    process.wait(timeout=300)  # 5 minute timeout per test method
+
+                    if process.returncode != 0:
+                        print(f"      ⚠️  {method} failed (exit code {process.returncode})")
+                        all_passed = False
+                    else:
+                        print(f"      ✓ {method} passed")
+
+                    # Kill lingering JSII processes between tests
+                    try:
+                        subprocess.run(["pkill", "-9", "-f", "jsii-java-runtime"],
```

Ruff (0.14.8)

Starting a process with a partial executable path

(S607)